spacelift/resource_policy.go

```diff
@@ -11,6 +11,25 @@ import (
 	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
 )
 
+var policyTypes = []string{
```

Comment:
Please add a comment about what's happening here. Why there are two access policies, why there's the type name replacements, and how they interact.

It took me a while of thinking to understand what's happening here.

Reply:
Done.